python_modules/libraries/dagster-fivetran/dagster_fivetran/resources.py

```diff
@@ -1179,10 +1191,10 @@ class FivetranWorkspaceDefsLoader(StateBackedDefinitionsLoader[Mapping[str, Any]
     def defs_key(self) -> str:
         return f"{FIVETRAN_RECONSTRUCTION_METADATA_KEY_PREFIX}/{self.workspace.account_id}"
 
-    def fetch_state(self) -> FivetranWorkspaceData:  # pyright: ignore[reportIncompatibleMethodOverride]
-        return self.workspace.fetch_fivetran_workspace_data()
+    def fetch_state(self) -> FivetranWorkspaceData:
+        return self.workspace._fetch_fivetran_workspace_data()  # noqa
```

Comment:
I'd much rather this method gets moved off of `FivetranWorkspace` and into `FivetranWorkspaceDefsLoader` -- that'd get rid of this private method access and also be a bit clearer

Reply:
Very fair - done in [f290702](https://github.com/dagster-io/dagster/pull/28983/commits/f29070278de11a8c65445f85505e93574270ecb8). I will do the same for dbt Cloud where we have the same pattern.